src/sentry/static/sentry/app/views/settings/components/forms/form.tsx

```diff
@@ -67,17 +68,6 @@ export default class Form extends React.Component<Props> {
     form: PropTypes.object.isRequired,
   };
 
-  static defaultProps = {
-    cancelLabel: t('Cancel'),
-    submitLabel: t('Save Changes'),
-    submitDisabled: false,
-    submitPriority: 'primary' as 'primary',
-    className: 'form-stacked',
-    requireChanges: false,
-    allowUndo: false,
-    saveOnBlur: false,
-  };
```

Comment:
Don't think we _explicitly_ had to remove this, but we didn't have it typed correctly, so it could be problematic.